### ISSUE TITLE
Release 0.5

### ISSUE THREAD

**Issue by dgyesbreghs** [Zewo/Data#3]
Can you please make a new release? Otherwise the swift package manager will clone from the wrong repo.
Expected result: https://github.com/open-swift/C7.git
Actual result: https://github.com/SwiftX/C7.git

**Comment by paulofaria** [comment]
ok, I'll do it. what module do you have that depends on this?

**Comment by paulofaria** [comment]
just a notice. this repo is kinda useless now. you should depend on `C7` directly. we already removed dependency from it in all our modules. you would be the only one using `0.5` haha.

**Comment by dgyesbreghs** [comment]
Vapor together with JSON (InterchangeData)

**Comment by paulofaria** [comment]
@dylangyesbreghs ok so why is the tag needed?

**Comment by dgyesbreghs** [comment]
@paulofaria i guess InterchangeData needs also an update on the Package file (just created a PR) and a new release tag...
https://www.dropbox.com/s/367mfnnbnojrejv/Screenshot%202016-04-12%2014.00.43.png?dl=0
We are getting this error message, because most modules are using https://github.com/open-swift/ some are using https://github.com/SwiftX...

PR: https://github.com/Zewo/StructuredData/pull/2

**Comment by paulofaria** [comment]
@dylangyesbreghs I can't read dutch? is that dutch? haha. ok. I thought that was already fixed. And I didn't Vapor depended on `InterchangeData`. only knew about `JSON`. is this new?

**Comment by dgyesbreghs** [comment]
@paulofaria sorry wrong screenshot :p
https://www.dropbox.com/s/xreuklw6jkc7i6z/Screenshot%202016-04-12%2013.58.25.png?dl=0

**Comment by dgyesbreghs** [comment]
@paulofaria  Yeah no, vapor only depends on JSON, but JSON depends on InterchangeData ;)
